src/venue/core.cljs

```diff
@@ -120,6 +120,13 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+(defn- init-services
+  []
+  (doseq [service (:services @state)]
+    (let [handler ((val service))]
+      (when (satisfies? IInitialise handler)
```

Comment:
I want to talk about this more. I added a comment to the issue you raised regarding view-models using `reify` which is philosophically identical to this: https://github.com/MastodonC/venue/issues/5